Sources/ArcGISToolkit/Components/Offline/OnDemand/OnDemandConfigurationView.swift

```diff
@@ -18,7 +18,7 @@ import SwiftUI
 /// A view that can provides a configuration for taking an on-demand area offline.
 struct OnDemandConfigurationView: View {
     /// The online map.
-    @State private(set) var map: Map
```

Comment:
What is the point of this change? Without this `@State` we were seeing a bug where the mapview would reset